tensorflow/lite/tools/cmake/modules/xnnpack/CMakeLists.txt

```diff
@@ -0,0 +1,59 @@
+#
+# Copyright 2021 NXP. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# The "-mcpu=" switch might be introduced externaly into CMake: either in <LANG>_FLAGS or
```

Comment:
I wonder if we don't need to touch "-march="?

Reply:
Not necessary, XNNPACK will add the `-march=` flag on its own, last one applies. With `-mcpu` the problem is that XNNPACK does not use this flag, therefor the one from the environment will apply.